models/iaf_psc_alpha_canon.h

```diff
@@ -115,31 +123,43 @@ if tau_m is equal to tau_syn, to avoid numerical instabilities.
 For details, please see doc/model_details/IAF_neurons_singularity.ipynb.
 
 A further improvement of precise simulation is implemented in iaf_psc_exp_ps
-based on [3].
+based on [3]_.
 
 For details about exact subthreshold integration, please see
-``doc/model_details/exact-integration.ipynb``.
+<https://github.com/nest/nest-simulator/blob/master/doc/model_details/exact-integration.ipynb>`_
+in the NEST source code (docs/model_details).
```

Comment:
```suggestion
in the NEST source code (doc/model_details).
```

Reply:
Actually this link looks weird in the rendered version. I just copied the one from [models/iaf_psc_alpha.h#L95](https://github.com/nest/nest-simulator/blob/master/models/iaf_psc_alpha.h#L95), but that might already be looking strange. can you please have a look?